### ISSUE TITLE
webapp archetype is broke

### ISSUE THREAD

**Issue by buehner** [terrestris/shogun2#335]
The webapp archetype is not working as expected. We should get rid of it or fix it.

**Comment by marcjansen** [comment]
butvwe tested it durin ci right?

**Comment by dnlkoch** [comment]
No, currently the artifact is not tested in CI, but we should definitely adjust this.

**Comment by kidac** [comment]
is following error message related to this issue?

`mvn archetype:generate -DarchetypeCatalog=local -DarchetypeGroupId=de.terrestris -DarchetypeArtifactId=shogun2-webapp-archetype`

`[INFO] --- maven-archetype-plugin:3.1.2:generate (default-cli) @ shogun-core ---
[INFO] Generating project in Interactive mode
[WARNING] Specified archetype not found.
[INFO] No archetype defined. Using maven-archetype-quickstart (org.apache.maven.archetypes:maven-archetype-quickstart:1.0)
Choose archetype:
   Your filter doesn't match any archetype (hint: enter to return to initial list)
Choose a number or apply filter (format: [groupId:]artifactId, case sensitive contains): :`

**Comment by hwbllmnn** [comment]
Actually, the artifact is called differently nowadays, try

```mvn archetype:generate -DarchetypeCatalog=local -DarchetypeGroupId=de.terrestris -DarchetypeArtifactId=shogun-core-webapp-archetype -DarchetypeVersion=4.0.0```

Still, I'm currently unsure whether the archetype is in a working state.
